src/include/kdump/uifunctions.rb

```diff
@@ -1468,15 +1468,16 @@ def HandleFADump(_key, event)
       UI.ChangeWidget(Id(:auto_resize), :Enabled, !use_fadump_value)
 
       update_usable_memory
-      refresh_kdump_memory
+      refresh_kdump_memory(use_fadump_value)
 
       nil
     end
 
-    def refresh_kdump_memory
+    def refresh_kdump_memory(fadump)
```

Comment:
NP: I'd use _value_ instead of _fadump_ for this variable name.

Reply:
I am not sure if it will be better. It is boolean that says if kdump memory is computed with or without firmware assistence ( fadump ).